### PR TITLE
fix: tpyo in ConfidentialStore type

### PR DESCRIPTION
**Pull request by sambarnes**
## 📝 Summary

was taking [notes on the diff](https://sambarnes.github.io/suave-geth-diff/) and saw this

feel free to close if ya wanna resolve yourself :)

---

* [x] I have seen and agree to CONTRIBUTING.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ graph TB
     classDef lightgreen fill:#b3c69f,stroke:#444,stroke-width:2px, color:#333;
 ```
 
-The capabilities enabled by this modified runtime are exposed via the APIs `ConfiendialStoreBackend` , `MempoolBackend`, `ConfiendialStoreBackend`, as well as access to `confidentialInputs` to confidential compute requests and `callerStack`.
+The capabilities enabled by this modified runtime are exposed via the APIs `ConfidentialStoreBackend` , `MempoolBackend`, `ConfidentialStoreBackend`, as well as access to `confidentialInputs` to confidential compute requests and `callerStack`.
```
Comment (sambarnes) [inline README.md:150]: 🤔 

```suggestion
The capabilities enabled by this modified runtime are exposed via the APIs `ConfidentialStoreBackend` , `MempoolBackend`, `OffchainEthBackend`, as well as access to `confidentialInputs` to confidential compute requests and `callerStack`.
```

**Comment by Ruteri** [comment]
Love the tpyo in PR description :)

**Comment by Ruteri** [comment]
@sambarnes can you adjust for the linter? Alterantively you can give me write access to your repo I'll fix it there (I don't want to take over your PR)

**Comment by sambarnes** [comment]
aye aye -- should be solid now @Ruteri 

cheers :)

**Review by Ruteri — APPROVED** [review]

**Comment by Ruteri** [comment]
:pray:

**Pull request merged**